internal/connector/test/integration/cucumber_steps.go

```diff
@@ -114,7 +115,7 @@ func (s *extender) deleteKeycloakClients(sc *godog.Scenario, err error) {
 
 		for _, clientID := range clientIds {
 			if err := keycloakService.DeleteServiceAccountInternal(clientID); err != nil {
-				glog.Errorf("Error deleting keycloak client with clientId %s: %s", clientID, err)
+				glog.Errorf("error deleting keycloak client with clientId %s: %s", clientID, err)
```

Comment:
This does not needs to be changed (see previous comment)

Reply:
thanks @ziccardi . Should be addressed now.

Reply:
@valeriiashapoval There are still several logs that have been lowercased.

Reply:
thanks @ziccardi . all logs should be fixed now.